Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDeviceRef.cpp

```diff
@@ -0,0 +1,78 @@
+//
+//  plMetalDeviceRef.cpp
+//  CoreLib
+//
+//  Created by Colin Cornaby on 11/10/21.
+//
```

Comment:
I don't think this file should even exist. It was from a far earlier version of the renderer. Nice catch. It would be reasonable to rebase this out - but for the sake of review I'll just remove it from ToT for now.

Reply:
I also want to add plMetalDeviceRefs.cpp and plMetalDeviceRef.h (singular) is inherited from the DX pipeline. I don't think thats been fixed since the time this branched.